package/cluster/services/composition.yaml

```diff
@@ -120,3 +142,25 @@ spec:
           toFieldPath: spec.forProvider.values.serviceAccount.customAnnotations[eks.amazonaws.com/role-arn]
           policy:
             fromFieldPath: Required
+        - type: ToCompositeFieldPath
+          fromFieldPath: metadata.name
+          toFieldPath: status.releaseNameUXP
+
+    - name: release-uxp-uses-xeks
+      base:
+        apiVersion: apiextensions.crossplane.io/v1alpha1
+        kind: Usage
+        spec:
+          of:
+            apiVersion: aws.platformref.upbound.io/v1alpha1
+            kind: XEKS
+          by:
+            apiVersion: helm.crossplane.io/v1beta1
+            kind: Release
```

Comment:
Is it a direct dependency on `Release` MR, not XServices XR to avoid Foreground deletion style, correct?

Reply:
Yes, I may test it against the Composite again with foreground deletion enabled.

Reply:
I don't think it will work across Composites (without a claim). [defaultCompositeDeletePolicy](https://doc.crds.dev/github.com/crossplane/crossplane/apiextensions.crossplane.io/CompositeResourceDefinition/v1@v1.13.2#spec-defaultCompositeDeletePolicy) does not work on XRs that are created without a Claim (as in XServices), so `XService` gets deleted immediately and takes the `Usage` and `XEKS` with it.

Reply:
Setting `compositeDeletePolicy: Foreground` in the `XCluster` Claim seems to work with a usage between composites. I'll keep testing.